soccer/gameplay/skills/intercept.py

```diff
@@ -6,9 +6,15 @@
 
 # wraps up OurRobot.move() into a Skill so we can use it in the play system more easily
 class Intercept(single_robot_behavior.SingleRobotBehavior):
-    def __init__(self, pos=None):
+
+    #controls if the robot should face the ball while intercepting
+    faceBall = False
```

Comment:
This version of `faceBall` isn't being used anywhere. If you ever want to access this one, you generally have to do `Intercept.faceBall`. Most of the time for these class variables, we only ever use them as constants (See adaptive formation for example).

Reply:
I think this is the only issue left. 

The only time `goalie.py` is used is during penalty kicks (which is basically untested), does this seem like a good idea for penalty kicks too?

Reply:
I'm not really sure what change to make here. I was under the impression that I was using that variable when I use self.faceBall in line 49. I'm not sure where I can put it that will still give me access to that information.